### PR TITLE
Add zziplib package

### PR DESCRIPTION
**Pull request by compomega**
This library needs fnmatch built into newlib/libsysbase to link properly.

### CODE REVIEW

File: switch/zziplib/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+pkgname=switch-zziplib
+pkgver=0.13.69
+pkgrel=1
+pkgdesc='The ZZIPlib provides read access on ZIP-archives and unpacked data. It features an additional simplified API following the standard Posix API for file access'
+arch=('any')
+url='https://github.com/gdraheim/zziplib/'
+license=('LGPL')
+options=(!strip libtool staticlibs)
+source=("https://github.com/gdraheim/zziplib/archive/v${pkgver}.tar.gz")
+makedepends=('devkitA64' 'devkitpro-pkgbuild-helpers' 'switch-zlib')
```
Comment (WinterMute) [inline switch/zziplib/PKGBUILD]: switch-zlib should be in depends group, not makedepends.

File: switch/zziplib/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+groups=('switch-portlibs')
+
+build() {
+  cd zziplib-$pkgver
+
+  source /opt/devkitpro/devkita64.sh
+  source /opt/devkitpro/switchvars.sh
+
+  ./configure --prefix="${PORTLIBS_PREFIX}" --host=aarch64-none-elf \
+    --disable-shared --enable-static --with-zlib="${PORTLIBS_PREFIX}"
```
Comment (WinterMute) [inline switch/zziplib/PKGBUILD]: is the --with-zlib option necessary?

In theory it should just pick up zlib from the flags we set via switchvars.sh.

**Comment by compomega** [comment]
I've made those corrections.

**Comment by carstene1ns** [comment]
This is currently lacking a pgp signature in pacman database.

**Comment by WinterMute** [comment]
Fixed now, thanks for pointing it out.

**Pull request merged**